modules/ROOT/pages/_partials/installing-mobile-services.adoc

```diff
@@ -19,8 +21,8 @@ IMPORTANT: OpenShift oc executable must be located in a system location that is
 +
 IMPORTANT: Ensure that you can run link:https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#getting-started[oc cluster up, window="_blank"] with no errors before moving on to the installation of AeroGear.
 +
-
-* A link:https://hub.docker.com/[Docker Hub, window="_blank"] account
+* A link:https://hub.docker.com/[Docker Hub] account
+//** For Linux (RHEL), manage Docker as a non-root user. For more information, see section _Manage Docker as a non-root user_ in  link:https://docs.docker.com/install/linux/linux-postinstall/[Post-installation steps for Linux].
```

Comment:
true for fedora 27 and 28

Reply:
If you have tested this to be true, I'd create a new PR and make the update as this PR was specifically for RHEL.

Reply:
i see that this has just been merged, ok another issue/pr is it, then

Reply:
so, RHEL is mentioned

Reply:
It is, but it is commented out. I left his in as when (I'm an optimist) the Openshift bug is fixed, this will need to be put back in and I can see this being forgotten about.

Reply:
ah, ok, didn't see the commenting